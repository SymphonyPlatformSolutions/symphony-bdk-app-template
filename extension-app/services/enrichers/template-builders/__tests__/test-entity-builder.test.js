import TestEntityBuilder from '../test-entity-builder';

describe('TestEntityBuilder', () => {
  it('Should build the enriched message', () => {
    expect(TestEntityBuilder.build()).toMatchSnapshot();
  });
});
