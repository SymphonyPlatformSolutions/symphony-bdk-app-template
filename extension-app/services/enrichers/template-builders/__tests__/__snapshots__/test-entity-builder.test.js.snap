// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TestEntityBuilder Should build the enriched message 1`] = `
"<messageML>
  <card class=\\"barStyle\\" data-icon-src=\\"https://via.placeholder.com/50\\" data-accent-color=\\"tempo-bg-color--blue\\">
    <header>
      <div style=\\"margin-bottom: 12px;\\"><b>My enriched message</b></div>
      <div>Some other stuff to be said in the title</div>
    </header>

    <body>
      <div>Body content inside body</div>
    </body>
  </card>
</messageML>"
`;
